packages/blockchain-extension/test/commands/importNodesToEnvironmentCommand.test.ts

```diff
@@ -216,6 +216,22 @@ describe('ImportNodesToEnvironmentCommand', () => {
             logSpy.getCall(1).should.have.been.calledWith(LogType.SUCCESS, 'Successfully imported all nodes');
         });
 
+        // Update this test when doing issue for filtering nodes
+        it('should test nodes can be filtered for an OpsTool environment', async () => {
+            showEnvironmentQuickPickStub.resolves({ label: OpsToolRegistryEntry.name, data: OpsToolRegistryEntry });
+            await vscode.commands.executeCommand(ExtensionCommands.EDIT_NODE_FILTERS, undefined, false, undefined, true);
```

Comment:
Shouldn't all tests for Ops Tools in this file execute ExtensionCommands.EDIT_NODE_FILTERS instead of ExtensionCommands.IMPORT_NODES_TO_ENVIRONMENT ?